oeo-imports/uo/extract-uo-module.sh

```diff
@@ -2,21 +2,28 @@
 #curl -L https://data.bioontology.org/ontologies/UO/submissions/219/download?apikey=8b5b7825-538d-40e0-9e9e-5ab9274a9aeb > uo-full-download.owl
 curl -L http://purl.obolibrary.org/obo/uo.owl > uo-full-download.owl
 # Extract the terms we want with downward hierarchy
-robot merge --input uo-full-download.owl extract --method MIREOT --lower-terms uo-w-hierarchy.txt --intermediates all --output uo-extracted.owl
-## add xmlns:obo="http://purl.obolibrary.org/obo/"
-#sed -i 's/xmlns:owl/xmlns:obo="http:\/\/purl.obolibrary.org\/obo\/"\xmlns:owl/' uo-module-temp.owl
-# Replace "<rdfs:comment>" with "obo:IAO_0000115" 
-sed -i 's/<rdfs:comment>/<obo:IAO_0000115>/g' uo-extracted.owl
-## Replace "<rdfs:comment>" with "</obo:IAO_0000115>"
+robot merge --input uo-full-download.owl extract --method MIREOT --lower-terms uo-w-hierarchy.txt --intermediates all --output uo-temp-extracted.owl
+## add prefix for OBO xmlns:obo="http://purl.obolibrary.org/obo/"
+# Linux sed -i 's/xmlns:owl/xmlns:obo="http:\/\/purl.obolibrary.org\/obo\/"\xmlns:owl/' uo-module-temp.owl
+# windows
+(Get-Content -ReadCount 0 uo-temp-extracted.owl) -replace 'xmlns:owl', 'xmlns:obo="http://purl.obolibrary.org/obo/#"
```

Comment:
I reviewed your script again. I think this line causes the problem with the additional `#`. What happens if you delete it here?
```suggestion
(Get-Content -ReadCount 0 uo-temp-extracted.owl) -replace 'xmlns:owl', 'xmlns:obo="http://purl.obolibrary.org/obo/"
```

Reply:
Ok, seems to work. The prefix can live without the # 👍